source/administration/command-line-tools.rst

```diff
@@ -1596,6 +1596,7 @@ mattermost webhook
     -  `mattermost webhook delete`_ - Delete incoming and outgoing webhooks.
     -  `mattermost webhook list`_ - List all webhooks.
     -  `mattermost webhook modify-incoming`_ - Modify existing incoming webhook by changing its title, description, channel or icon url.
+    -  `mattermost webhook modify-outgoing`_ - Modify existing outgoing webhook by changing its title, description, channel, icon, url, content-type, and triggers.
```

Comment:
Should it be `an existing outgoing webhook` or `existing outgoing webhooks`?

Similar for the incoming webhook on the line above.

Reply:
Good catch @jasonblais.  Updated all the relevant places.